gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -155,6 +155,30 @@ def _get_sample_non_variant():
 
   return non_variant
 
+def _get_sample_variant_1_based():
```

Comment:
The only difference here is `start=1234`, instead of duplicating the whole method could we instead add an input flag to the 3 methods (`_get_sample_variant_*`)?